content/en/docs/v3.5/tutorials/how-to-conduct-elections.md

```diff
@@ -1,16 +1,24 @@
----
```

Comment:
We also need to add these changes into the `content/en/docs/v3.6/tutorials/how-to-conduct-elections.md` file as well.

Reply:
> We also need to add these changes into the `content/en/docs/v3.6/tutorials/how-to-conduct-elections.md` file as well.

OK.. will look for that one as well